examples/run/common/throughput_mt.ipp

```diff
@@ -190,6 +191,20 @@ int throughput_mt(std::string_view description, int argc, char* argv[],
                                          "Event processing"}
               << std::endl;
 
+    // Print results to log file
+    if (mt_cfg.log_result) {
+        std::ofstream logFile;
+        logFile.open(mt_cfg.log_file, std::fstream::app);
+        logFile << "\"" << throughput_cfg.input_directory << "\""
+                << "," << mt_cfg.threads << ","
+                << throughput_cfg.cold_run_events << ","
+                << throughput_cfg.cold_run_events << ","
+                << throughput_cfg.processed_events << ","
+                << times.get_time("Warm-up processing").count() << ","
+                << times.get_time("Event processing").count() << std::endl;
```

Comment:
I also considered directly printing out the events/s measurement but thought it might best to leave the output as "clean" as possible by leaving to the user to make the FP divisions. Also note that the output files don't have any headers. I could make this somewhat more complex by opening the file in read & write mode and first checking if the headers are already present, and if not write them out.